docker-build

```diff
@@ -38,22 +47,52 @@ docker_pull() {
 
 pull() {
   local repo="$1"
-  docker_pull "$repo:$BRANCH" || docker_pull "$repo:master" || docker_pull "$repo:latest" || echo "$repo"
+  docker_pull "$repo:$BRANCH" \
+    || docker_pull "$repo:master" \
+    || docker_pull "$repo:latest" \
+    || echo "$repo"
 }
 
 # Build builds the docker image and tags it with the git sha and branch.
 build() {
-  local repo="$1" cache="$2"
+  local repo="$1"
+  shift
+  args=""
   if [ "$extra_tags" = true ];
     then
-      tag_flags="-t $repo -t $repo:$SHA -t $repo:$BRANCH"
+      args="-t $repo -t $repo:$SHA -t $repo:$BRANCH"
   elif [ "$BRANCH" = "master" ];
     then
-      tag_flags="-t $repo -t $repo:$BRANCH"
+      args="-t $repo -t $repo:$BRANCH"
+  else
+      args="-t $repo"
+  fi
+  if [ "$DOCKER_BUILDKIT" = "1" ]; then
+    args="$args --build-arg BUILDKIT_INLINE_CACHE=1 --progress plain"
```

Comment:
Bash supports `+=` operator

Reply:
Oh, I keep forgetting this isn't bash, it's alpine's ash... I don't think `+=` is supported unfortunately.

Reply:
This is why I added `shellcheck` haha